app/views/notifications/_follow.html.erb

```diff
@@ -1,44 +1,36 @@
   <% if notification.json_data["aggregated_siblings"].length == 1 %>
-    <% cache "activity-follow-button-#{first_notification['path']}-#{first_notification['name']}" do %>
-      <a href="<%= first_notification["path"] %>"><%= first_notification["name"] %></a> followed you!
-      <%= follow_button(notification.decorate.mocked_object("user"), "follow-back") %>
+    <% first_notification = notification.json_data["aggregated_siblings"].first %>
+    <% cache "activity-profile-pic-#{first_notification['id']}-#{first_notification['profile_image_90']}" do %>
+      <a href="<%= first_notification["path"] %>" class="crayons-avatar <% if notification.json_data["aggregated_siblings"].length == 1 %>crayons-avatar--xl<% else %>crayons-avatar--l<% end %> shrink-0">
+        <img src="<%= first_notification["profile_image_90"] %>" class="crayons-avatar__image" alt="link to <%= first_notification["username"] %>'s profile" width="48" height="48">
+      </a>
     <% end %>
-  <% elsif notification.json_data["aggregated_siblings"].length == 2 %>
-    <% json_data_array = notification["json_data"]["aggregated_siblings"] %>
-    <div class="follower-pic-row">
-      <div class="tiny-pic">
-        <a href="<%= json_data_array.last["path"] %>">
-          <img src="<%= json_data_array.last["profile_image_90"] %>" alt="link to <%= json_data_array.last["name"] %>'s profile" class="round">
-        </a>
-      </div>
-    </div>
-    <br>
-    <div class="notification-message">
-      <a href="<%= json_data_array.first["path"] %>"><%= json_data_array.first["name"] %></a> and
-      <a href="<%= json_data_array.second["path"] %>"><%= json_data_array.second["name"] %></a> followed you!
+    <div class="notification__content">
+      <% cache "activity-follow-button-#{first_notification['path']}-#{first_notification['name']}" do %>
+        <div class="flex justify-between items-center">
+          <p><a href="<%= first_notification["path"] %>" class="crayons-link fw-bold"><%= first_notification["name"] %></a> followed you!</p>
+          <%= follow_button(notification.decorate.mocked_object("user"), "follow-back") %>
+        </div>
+      <% end %>
     </div>
   <% else %>
-    <div class="follower-pic-row">
-      <% notification.json_data["aggregated_siblings"][1..10].each do |sibling| %>
-        <div class="tiny-pic">
-          <a href="<%= sibling["path"] %>">
-            <img src="<%= sibling["profile_image_90"] %>" alt="link to <%= sibling["name"] %>'s profile" class="round">
-          </a>
-        </div>
+    <% json_data_array = notification["json_data"]["aggregated_siblings"] %>
+    <div class="flex mb-4">
+      <% notification.json_data["aggregated_siblings"][0..10].each do |sibling| %>
+        <a href="<%= sibling["path"] %>" class="crayons-avatar crayons-avatar--l shrink-0 mx-1">
```

Comment:
Since a lot of the other avatars are `crayons-avatar--xl`, I think that this one would look more uniform if it's also using `crayons-avatar--xl`.

Reply:
It is intentionally smaller because it's for scenario when we show aggregated new followers (for example 5 ppl followed you at the same time) - we then gonna show 5 avatars in the row, in one card. That's why it's smaller - with more `xl` avatars next to each other, this felt quite overwhelming.

Reply:
Okay thanks!